_includes/fix_analytics_signup_form.html

```diff
@@ -0,0 +1,352 @@
+        </div>
+    </form>
+    <div
+        class="contact-form-overlay contact-form-overlay-clickthrough"
+        data-analytics-form-overlay
+    >
+        {% if include.language == "danish" %} Sender... {% else %} Sending... {% endif %}
+    </div>
+</section>
+
+<script>
+    (function () {
+        var form = document.getElementById("contact-form-analytics");
+        var honeypot = form.UZZvvqxJKRlGGbvYaVxz;
+        var nameField = form.name;
+        var emailField = form.email;
+        var currencyField = form.currency;
+        var localeField = form.locale;
+        var customerTypeField = form.customer_type;
+        var locationStreetField = form.location_street;
+        var locationCityField = form.location_city;
+        var locationZipField = form.location_zipcode;
+        var locationStateField = form.location_state;
+        var locationCountryIdField = form.location_country_id;
+        var phoneField = form.phone;
+        var registrationNumberField = form.registration_number;
+        var voucherField = form.voucher;
+        var acceptTermsField = form.accept_terms;
+        var domainField = form.domain_1;
+
+        var contactFormOverlay = document.querySelector("[data-analytics-form-overlay]");
+        var addAdditionalDomainButton = document.querySelector("[data-add-additional-domain");
+        var domainsContainer = document.querySelector("[data-domains-container");
+
+        addAdditionalDomainButton.addEventListener("click", function (e) {
+            e.preventDefault();
+            addNewDomainField();
+        });
+
+        form.addEventListener("submit", function (e) {
+            e.preventDefault();
+            var honeypotValue = honeypot.value;
+            if (honeypotValue !== "") {
+                document.body.append("honeypot: '" + honeypotValue + "' (" + typeof honeypotValue + ")");
+                return;
+            }
+
+            showOverlay();
+
+            var customer = {
+                email: emailField.value,
+                currency: currencyField.value,
+                locale: localeField.value,
+                type: customerTypeField.value,
+                name: nameField.value,
+                countryId: locationCountryIdField.value,
+                street: locationStreetField.value,
+                cityText: locationCityField.value,
+                stateText: locationStateField.value,
+                zipcode: locationZipField.value,
+                phone: phoneField.value,
+                registrationNo: registrationNumberField.value,
+            };
+            var domainList = document.querySelectorAll("[data-domain-field]");
+            var domains = [];
+            var domainValues = domainList.forEach((input) => {
+                if (input.value) {
+                    domains.push(input.value);
+                }
+            });
```

Comment:
Ech I don't like NodeLists because they don't have filter and map, which is what you are doing here...  I guess I'll let it pass.

Generally we have tried avoiding `() => {}` lambda functions and use `function() {}` instead :-) 

I think it may be overly cautious with current browser support... but it's what we've done elsewhere